### PR TITLE
FreeFem: new version 4.10

### PR DESCRIPTION
**Pull request by loumalouomega**
Besides, for the moment the serial installation does not work. Working on it

### CODE REVIEW

**Comment by corentin-dev** [comment]
Is there a reason the archive changed from `tar` to `zip` ? I'm just being curious.

**Comment by loumalouomega** [comment]
> Is there a reason the archive changed from `tar` to `zip` ? I'm just being curious.

In fact not, I can revert the tar. That would explain why I needed to update the checksums in order to work...

**Comment by corentin-dev** [comment]
Probably yes, can you revert it please?

Thank you !

**Comment by loumalouomega** [comment]
> Probably yes, can you revert it please?
> 
> Thank you !

Sorry for the inconvenience

**Comment by corentin-dev** [comment]
No problem! I'll quickly review the changes and let you know if it is OK for me.

File: var/spack/repos/builtin/packages/freefem/package.py [hunk 1]
```diff
@@ -15,10 +15,11 @@ class Freefem(AutotoolsPackage):
     """
 
     homepage = "https://freefem.org"
-    url      = "https://github.com/FreeFem/FreeFem-sources/archive/refs/tags/v4.9.tar.gz"
+    url      = "https://github.com/FreeFem/FreeFem-sources/archive/refs/tags/v4.10.tar.gz"
```
Comment (sethrj) [inline var/spack/repos/builtin/packages/freefem/package.py:18]: This isn't necessary (for future reference): the url is just a template.

**Review by sethrj — APPROVED** [review]

File: var/spack/repos/builtin/packages/freefem/package.py (hunk 1 above)
Comment (loumalouomega) [reply var/spack/repos/builtin/packages/freefem/package.py:18]: Okay

Comment (loumalouomega) [reply var/spack/repos/builtin/packages/freefem/package.py:18]: It is possible to define different url? Just curious

**Pull request merged**